llama-index-networks/examples/demo/README.md

```diff
@@ -0,0 +1,96 @@
+# LlamaIndex Networks Demo
+
+In this demo, we have 3 network contributor services, whose Python source
+codes can be found in the folders listed below:
+
+- contributor-1/
+- contributor-2/
+- contributor-3/
+
+Once all of these services are up and running (see usage instructions
+attached below), then we can connect to them using a `NetworkQueryEngine`.
+The demo for doing that is found in the notebook: `network_query_engine.ipynb`.
+
+## Building and Running the 3 Network Contributor Services
+
+### Setup Environment Variables
+
+Each of the three Contributor Services wrap `QueryEngine`'s that utilize
+OpenAI as the LLM provider. As such, you'll need to supply an `OPENAI_API_KEY`.
+
+To do so, we make use of .env files. Each contributor folder requires a filled
+in `.env.contributor.service` file. You can use the `template.env.contributor.service`,
+fill in your openai-api-key and then save it as `.env.contributor.service`.
```

Comment:
Is there any reason to not just have the .env in each contributor? This tripped me up slightly.

Also, I guess contributor_2 depends on a data folder I have to make myself?

Reply:
Ah, yeah we can make it simpler. The `ContributorClientSettings` and `ContributorServiceSettings` will read from `.env` file if `.env.contributor.service`  doesn't exist. So we can use just `.env` and it should work.

Reply:
Re: the data... sorry. Yah, I just pushed the data. I think that's easier than having the user create the data folder themselves. I've pushed up two data files:

- contributor-1/data/paul_graham_essay.txt
- contributor-2/data/arxiv.pdf (not the right name)

Reply:
thank you! Mostly just reporting my time as I try out the demo :)